runners/core-java/src/main/java/org/apache/beam/runners/core/metrics/ExecutionStateTracker.java

```diff
@@ -304,7 +304,7 @@ public long getNextLullReportMs() {
     return nextLullReportMs;
   }
 
-  void takeSample(long millisSinceLastSample) {
+  public void takeSample(long millisSinceLastSample) {
```

Comment:
This is now public, consider add a javadoc

Reply:
[f371c80](https://github.com/apache/beam/pull/29592/commits/f371c80040c44c16e3ffafa0af2c2e9c1c5e9067)